### DISCUSSION TITLE
Should styling on Table Headers be reflected in the first column when using -Transpose option?

### DISCUSSION THREAD

**Discussion by rloone** [EvotecIT/PSWriteHTML#309] [Q&A]
I have a table that is a dataset with 1 header row and 1 data row.  The header has a different colour/background combo.

But the table has quite a few columns and I would like to transpose it.  When I do, I lose the styling of the headers.

Is this by design and should I be looking at something like conditional cell formatting to get around it?

Thanks

**Comment by PrzemyslawKlys** [comment]
Please provide some example.
